handler.go

```diff
@@ -15,7 +16,7 @@ func Handler() http.Handler {
 		var seconds int
 		if s := r.URL.Query().Get("seconds"); s == "" {
 			seconds = 30
-		} else if _, err := fmt.Sscanf(s, "%d", &seconds); err != nil || seconds <= 0 {
+		} else if seconds, err := strconv.Atoi(s); err != nil || seconds <= 0 {
```

Comment:
🐞 This introduces a subtle bug since there are now two variables called `seconds`, the one declared on line 16, and the one declared on this line which goes out of scope after the `if` block ends.

As a result the outer `seconds` always ends up `0` when explicitly requesting requesting e.g. `?seconds=123`, which essentially breaks that option.

Reply:
thanks @felixge for the quick review and pointing out the bug. Somehow missed the if scope. Will fix it and update.